src/pages/settings/Settings.jsx

```diff
@@ -29,7 +29,7 @@ function getInitialFormValues(schemas, data) {
 export default function Settings() {
   useDocumentTitle('SETTINGS_AND_PRIVACY');
 
-  const { data, refresh } = useGetMe();
+  const { data } = useGetMe();
```

Comment:
No invalidate statements in this file?

Reply:
Same with a few of the subsequent modified files.

Reply:
no, so the invalidate statements go in the hooks that update resources, and as a result the `refresh` commands are no longer needed in the components themselves